lib/data/master.toml

```diff
@@ -472,12 +472,12 @@
     [codes.naics.21111]
     name = "Oil and Gas Extraction"
 
-    [codes.naics.211111]
-    name = "Crude Petroleum and Natural Gas Extraction"
+    [codes.naics.211120]
+    name = "Crude Petroleum Extraction"
     sic_correlations = ["1311"]
 
-    [codes.naics.211112]
-    name = "Natural Gas Liquid Extraction"
+    [codes.naics.211130]
+    name = "Natural Gas Extraction"
     sic_correlations = ["2819", "1321"]
```

Comment:
I think you'll need to find codes.sic.2819 and add the naics_correlation to it.  These are further down in master.toml